common/serializers.py

```
@@ -0,0 +1,43 @@
+
+from rest_framework.request import Request
+from rest_framework.serializers import ModelSerializer, Serializer
+
+
+class DynamicFieldsModelSerializer(ModelSerializer):
+    """
+    A ModelSerializer that takes an additional `fields` argument that
+    controls which fields should be displayed.
+    """
+
+    def __init__(self, *args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)